plugins/generator-1.15.2/forge-1.15.2/templates/plant.java.ftl

```diff
@@ -436,11 +435,14 @@ import net.minecraft.block.material.Material;
 					return this.isValidGround(blockstate, worldIn, blockpos)
 				<#elseif data.plantType == "growapable">
 					<#if hasCondition(data.placingCondition)>
-					World world = worldIn.getDimension().getWorld();
-					int x = pos.getX();
-					int y = pos.getY();
-					int z = pos.getZ();
-					boolean additionalCondition = <@procedureOBJToConditionCode data.placingCondition/>;
+					boolean additionalCondition = true;
+					if (worldIn instanceof IWorld) {
+						IWorld world = (IWorld) worldIn;
+						int x = pos.getX();
+						int y = pos.getY();
+						int z = pos.getZ();
+						additionalCondition = <@procedureOBJToConditionCode data.placingCondition/>;
+					}
 					</#if>
 
 					Block block = blockstate.getBlock();

```

Comment:
Consider renaming this block to something else too as a "block" variable is already a field for block instance (`public static final Block block = null;`), so we don't shadow variables. Make sure to update all use cases of this so we don't accidentally reference the other block :)